test/fixtures/compound14.css

```diff
@@ -0,0 +1,31 @@
+
+:root {
+	--color: #ffffff;
+	--color2: #ffffff;
+	--color3: #ffffff;
+}
+
+#foo li {
+	color: var(--color);
+}
+
+#foo.bar ul:hover {
+	--color: #000000;
+	color: var(--color);
+}
+
+#bar li {
+	color: var(--color2);
+}
```

Comment:
It looks like the `#foo` and `#bar` cases can be split into separate tests without affecting what we're trying to test. And makes them easier to understand when they fail.